lib/pipe/csv.js

```diff
@@ -0,0 +1,110 @@
+const path = require("path");
+const fs = require("fs");
+const csvWriter = require('csv-writer');
+const { v4: uuidv4 } = require('uuid');
+const { isSameTest } = require('../util');
+const chalk = require('chalk');
+
+class CsvPipe {
+    #exportDir = process.env.EXPORT_DIR || "export"; //TODO: QUESTION: How create .env var?... and then should be removed
+    #csvSuffix = process.env.EXPORT_CSV_SUFFIX || "testomat.csv"; //TODO: should be removed
```

Comment:
```suggestion
    #csvFilePath = process.env.TESTOMATIO_CSV_FILE || "testomat.csv"; //TODO: should be removed
```

I would use only `TESTOMATIO_CSV_FILE` as a configuration

* with `TESTOMATIO_` prefix to distinguish that from other env variable
* a relative path to file so it could be executed as:

```
TESTOMATIO_CSV_FILE=export/report.csv npx codeceptjs run
```